input/fsh/QuestionnaireResponseSEVendorLite.fsh

```diff
@@ -0,0 +1,59 @@
+Profile: QuestionnaireResponseSEVendorLite
+Parent: QuestionnaireResponse
+Description: "Questionnaire response with restrictions for swedish vendors."
```

Comment:
make the status as 'draft' for  2 profiles?

Reply:
Meeting decision was no, no need for draft statuses at this time